src/containers/TodoList/index.js

```diff
@@ -0,0 +1,46 @@
+import React, { PropTypes } from 'react';
+
+import TodoBar from '../../components/TodoBar';
+import Todo from '../../components/Todo';
+
+const TodoList = ({ todos, toggleComplete, deleteCompleted, deleteTodo, handleFilterChange, filter }) => {
+  return (
+    <div>
+      <TodoBar deleteCompleted={deleteCompleted} handleFilterChange={handleFilterChange} />
+      {todos
+        .filter(todo => {
+          switch (filter) {
+            case 'none':
+              return todo;
```

Comment:
In your switch statement, you don't have to use `break` if you `return` from the condition.